resources/wiki-overrides.properties

```diff
@@ -62,13 +53,9 @@ vaddy-plugin=https://wiki.jenkins-ci.org/display/JENKINS/VAddy+Plugin
 violations=https://wiki.jenkins-ci.org/display/JENKINS/Violations
 xpdev=https://wiki.jenkins-ci.org/display/JENKINS/XP-Dev+Plugin
 
-# Workaround for INFRA-922
-absint-astree=https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=99910545
```

Comment:
https://github.com/jenkinsci/absint-astree-plugin/commit/3a03a5b334325cba2018093ec77fed6ffd2bd8c2 references the same URL.